runtime/tchannel_interfaces.go

```diff
@@ -36,7 +36,7 @@ type RWTStruct interface {
 type TChannelClient interface {
 	// Call should be passed the method to call, headers and the request/response thriftrw structs.
 	// The arguments returned are (whether there was an application error, unexpected error)
-	Call(ctx context.Context, service, method string, reqHeaders map[string]string, req, resp RWTStruct) (success bool, respHeaders map[string]string, err error)
+	Call(ctx context.Context, clientMethod, thriftService, thriftMethod string, reqHeaders map[string]string, req, resp RWTStruct) (success bool, respHeaders map[string]string, err error)
```

Comment:
We should not add a `clientMethod` argument, `clientMethod` is a concept of upper level abstraction, and has nothing to do with the `Call` interface.

Reply:
See previous comment.